front/components/DataSourceViewResourceSelectorTree.tsx

```diff
@@ -80,6 +86,7 @@ function DataSourceViewResourceSelectorChildren({
   showExpand,
   viewType = "documents",
 }: DataSourceResourceSelectorChildrenProps) {
+  const { plan, dustClientFacingUrl } = useContext(AssistantBuilderContext);
```

Comment:
that being said, becareful, i think the same component is used in vaults datasourceview selection (without the context).
You might want to test.

Reply:
You right good catch, will fix. Thanks for the heads up 👍🏼